modules/virt-booting-efi-mode.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt-efi-mode-for-vms.adoc
+
+[id="virt-booting-vms-efi-mode_{context}"]
+= Booting virtual machines in EFI mode
+
+Booting a virtual machine in EFI mode allows for a quicker boot process.
+
+[NOTE]
+====
+{VirtProductName} requires System Management Mode (`SMM`) to be enabled for Secure Boot in EFI mode to occur.
+====
+
+.Procedure
+To boot a VM in EFI Mode:
+
+. Create a YAML file that defines a VM object or a Virtual Machine Instance (`VMI`) object. Use the firmware stanza of the example YAML file:
++
+.Booting in `EFI` mode with secure boot active
+[source,yaml]
+----
+  apiversion: kubevirt.io/v1
+  kind: VirtualMachineInstance
+  metadata:
+    labels:
+      special: vmi-secureboot
+    name: vmi-secureboot
+  spec:
+    domain:
+      devices:
+        disks:
+        - disk:
+            bus: virtio
+          name: containerdisk
+      features:
+        acpi: {}
+        smm:
+          enabled: true
```

Comment:
I would add callouts on this line and the `secureBoot: true` line, to point out that you should edit these.